plugins/cpp_metrics/parser/include/cppmetricsparser/cppmetricsparser.h

```diff
@@ -21,14 +24,26 @@ namespace parser
 class CppMetricsParser : public AbstractParser
 {
 public:
+  static constexpr double LOC_SCALING = 1e+4;
```

Comment:
I'm guessing that this magic number is a multiplier to the actual metrics?

Reply:
Normally, the lack of cohesion metric is a real value in the range [0,1] (or [0,2] for the HS variant), which can only be stored in our current `unsigned int` value field if we premultiply it by some constant. This is that constant, and I agree that it's fairly arbitrary. I actually raised this concern in the original issue, but for now we agreed to go with this approach (https://github.com/Ericsson/CodeCompass/issues/681#issuecomment-1831904646).

Reply:
@dbukki As discussed on the last meeting in December 2023, make this field a float or a double. It should also be *nullable* to handle extremal values. See `#pragma db null` for ODB.

Reply:
@mcserep The value is now a double. See https://github.com/Ericsson/CodeCompass/pull/688/commits/8d757aec85a6e8a20ca17b07436127f448a9eac7